client/README.md

```diff
@@ -219,7 +219,7 @@ creation of Zeth notes.
 # Deposit 10 ether from `eth-address`, creating Zeth notes owned by Alice
 (env)$ zeth mix --out <public-zeth-address>,10 --vin 10
 ```
-where `<public-zeth-address>` is the contents of `zeth-address.json.pub`.
+where `<public-zeth-address>` is the contents of `zeth-address.pub`.
```

Comment:
content?

Reply:
I think either is correct in this situation.

Reply:
Actually, since we can just put the filename on the command line now I've updated the command and removed this line.